src/Smart.Design.Console/readme.md

```diff
@@ -0,0 +1,2 @@
+##CLI Design System
```

Comment:
- Please name the file in all caps: README.md
- Please add a space after the `#` (and maybe you only wanted one sharp ?)
- Please add a white line between each type of content (here, a title and a paragraph)
- Please write a complete sentence in the paragraph.
- I think it would be nice to provide a bit more information too.